organization/planning/plan-of-work-2023.md

```diff
@@ -0,0 +1,215 @@
+  * New language features
+  * New parallelism libraries as a potential long-term alternative to TBB
+    * E.g., HPX
+  * Key4Hep
+
+### Software Training
+
+* Switch to intermediate target audience and HEP-specific tools
+* Teach more scikit-HEP & IRIS-HEP packages
+  * Active community that is interested in growing its user base
+  * Use IRIS-HEP fellowships to expand training fellowships
+* Coordinate with new initiatives from the Research Software Engineering World
+* Be more open to materials of different formats and from different authors
+* State of Training Survey: identify blind-spots, start subgroups to work on them
+* Improve liaisons to experiments
+* Report experiences in Journals like JOSE
+* Hybrid training events
+
+* Expand Training Center
+  * Vision: One website as entry point for all HEP related training material
+
+* Concrete Events
+  * Carpentry workshops (still included in subscription):
+    * How can we lower attrition?
+    * How do we bundle carpentry intro together with our previous “last day” (if at all) → should probably be
+separate indico agenda
+  * Analysis preservation
+    * Second event in summer for scale?
+    * Now that all videos are in place, only need to find mentors for last day, so relatively easy
+  * ML + GPU
+    * Already have material, so should be somewhat easy to set up
+
+### Software Tools and Packaging
+
+* Continue Spack Discussions with contributions from Spack Developers and Experiments
+  * Update on EESSI
+  * Python packaging
+* Together with other Working Groups:
+  * C++ language features
+  * Spack training material
+
+### Data Analysis
+
+* Data compression
+  * several speakers and novel areas identified
+* Open data
+  * LHCb now has significant open data set
+  * Key for AGC, can serve as common ground for “challenges”
+* End user survey of data analysis approaches across experiments
+  * Needs careful design & large representative set of responses
+  * No recent comprehensive results available -> good to get overview of field
+* Set up a “Systematics challenge”
+  * Small analysis task to probe handling of systematic uncertainties
+  * Needs careful design (scope, effort involved to contribute)
+  * Would naturally frame further discussions on this topic following AEWII
+
+### Reconstruction and Software Triggers
+
+* Hot topics?
+  * 2021 survey showed 4D reco and heterogeneous computing - still relevant today
+* New topics
+  * Open data initiatives: Open data for reco studies (open thread by former conveners)
+  * Biased triggering and missed physics:
+    * Experiment TDAQ design and trigger/reco biases
+    * Strategies to “leave no stone unturned”
+  * Sustainability: Impact, indicators, better practices
+  * Focus on small/medium-sized experiments, e.g. Common DAQ/software frameworks
+  * Lessons learned” from new/upgraded experiments after 1-2 years of data-taking
+
+* Have a kick-off meeting in April (26?)
```

Comment:
the kick-off meeting is confirmed on the 26th of April, [indico](https://indico.cern.ch/event/1273894/).

Reply:
Thanks! Updated